Services/CO.CDP.OrganisationInformation.Persistence/Organisation.cs

```diff
@@ -15,7 +15,7 @@ public class Organisation : IEntityDate
     public IList<Identifier> Identifiers { get; set; } = [];
     public ICollection<OrganisationAddress> Addresses { get; set; } = [];
     public ICollection<ContactPoint> ContactPoints { get; set; } = [];
-    public List<PartyRole> Roles { get; init; } = [];
+    public List<PartyRole> Roles { get; set; } = [];
```

Comment:
If you're not replacing the collection, it doesn't need to be `set`table anymore right?

Reply:
Done